### ISSUE TITLE
Can TODS be installed in Windows environment?

### ISSUE THREAD

**Issue by kirathes** [datamllab/tods#97]
Can TODS be installed in Windows environment using command such like'pip install TODS'?

**Comment by MRTCc** [comment]
No. One of the main contributors has clearly stated that TODS is not Windows-compatible in this [issue](https://github.com/datamllab/tods/issues/104#issuecomment-1608207509)
